vmpooler.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'pickup', '~> 0.0.11'
   s.add_dependency 'puma', '~> 4.3'
   s.add_dependency 'rack', '~> 2.2'
-  s.add_dependency 'rake', '~> 12.3'
+  s.add_dependency 'rake', '>= 12.3', '< 14.0'
```

Comment:
It's a little strange to me that it did this range instead of `~> 13.0` like in the `Gemfile` but it's equivalent so maybe it's fine?